contracts/strategies/tetu-v1/TetuV1SingleTokenStrictStrategy.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "@tetu_io/tetu-contracts-v2/contracts/strategy/StrategyStrictBase.sol";
+import "@tetu_io/tetu-contracts-v2/contracts/interfaces/ITetuLiquidator.sol";
+import "../../integrations/tetu-v1/ISmartVault.sol";
+import "../../tools/AppLib.sol";
+import "../../tools/ERC20Helpers.sol";
+
+/// @title Simple auto compounding strategy for TETU V1 vaults.
+/// @author AlehNat
+contract TetuV1SingleTokenStrictStrategy is StrategyStrictBase, ERC20Helpers {
+  using SafeERC20 for IERC20;
+
+  string public constant override NAME = "TetuV1 Single Token Strict Strategy";
+  string public constant override PLATFORM = "TETU";
+  string public constant override STRATEGY_VERSION = "1.0.0";
+
+  uint private constant _ASSET_LIQUIDATION_SLIPPAGE = 500; // 0.5%
```

Comment:
for reward liquidation 5% slippage is fine, 0.5 too strict :)

Reply:
Updated